pytest_datafiles.py

```diff
@@ -5,6 +5,8 @@
 from py import path  # pylint: disable=E0611
 import pytest
 
+def pytest_configure(config):
+    config.addinivalue_line("datafiles", "datafiles(path): path to tmp data files")
```

Comment:
```suggestion
    config.addinivalue_line("markers", "datafiles(path): path to tmp data files")
```

As per docs: https://docs.pytest.org/en/latest/mark.html#registering-marks

Reply:
Right on.  Yes-Sir!  (Evoking memories of Jack Nicklaus winning the 1986 masters.)